client/starwhale/base/bundle_copy.py

```diff
@@ -444,3 +449,13 @@ def _do_upload_bundle_dir(
             raise
         else:
             self._do_ubd_end(upload_id=upload_id, url_path=url_path, ok=True)
+
+    @staticmethod
+    def _update_manifest(workdir: Path, patch: t.Dict[str, t.Any]) -> None:
+        file = workdir / DEFAULT_MANIFEST_NAME
+        # downloaded runtime is a tar file
+        if not os.path.exists(file):
```

Comment:
`if not os.path.exists(file):` --> `file.exists()` ?

Reply:
Done